### DISCUSSION TITLE
Help: Entangle error on each page that has a datatable

### DISCUSSION THREAD

**Discussion by isigillo** [rappasoft/laravel-livewire-tables#1936] [Q&A] [answered]
I find in console this error "Livewire Entangle Error: Livewire property ['tableName'] cannot be found on component ... " for each page of my application that has a datatable component.
The table mostly seems to work, but one or two tables shows weird behaviours. 

i am using

- livewire 3.5.6 
- laravel-livewire-tables 3.4.19
- tailwind 3.1
- vite 4.0
- laravel 10.48

**Comment by lrljoe** [comment]
This will be fixed in the next release (likely tomorrow).

**Reply by lrljoe** [reply, marked as answer]
Should be fixed:
## [v3.4.20] - 2024-09-10
### Bug Fixes
- Revert tableName to be public by @lrljoe in https://github.com/rappasoft/laravel-livewire-tables/pull/1937

**Reply by isigillo** [reply]
thank you
